### ISSUE TITLE
Feature request: Analyse for orphaned images too

### ISSUE THREAD

**Issue by binary64** [sverweij/dependency-cruiser#491]
### Context

We import images for our pages like this `import img from 'image.jpg'` and would love a way to detect unused images.

### Expected Behavior

The images should show as orphans and exit level >0

### Current Behavior

Images are ignored

### Possible Solution

Webpack sdk

**Comment by sverweij** [comment]
@binary64 thanks for this suggestion. It makes a lot of sense to include these, even though they're not strictly parseable. A similar case goes for `.json` which has been on my [wishlist](https://github.com/sverweij/dependency-cruiser/projects/1#card-31839559) for some time.

Making anything new a scannable extension by default would constitute a breaking change, though. So I've added a `extraExtensionsToScan` option where you can pass an array of extensions dependency-cruiser should look for when it scans your source tree (in `dependency-cruiser@10.3.0-beta-1`)

Install that beta and add an `extraExtensionsToScan` to the `options` section of your .dependency-cruiser.js and Bob should be your uncle:

```
  // ...
  extraExtensionsToScan: [
    ".jpg",
    ".png",
    ".webp",
    ".svg",
    ".json",
    // and any other extensions you want to consider
  ]
  // ...
```

(It's a beta because there might still be rough edges around this feature - so tweaks might still be forthcoming and automated tests need to be added).

=> could you let me know whether this works for you?

**Comment by binary64** [comment]
Woah, Bob is my uncle, how did you know!! This is amazing, thank you so much

I get this error:
```
binary64@DESKTOP MINGW64 /c/p/monorepo (dependency-cruiser)
$ yarn depcruise --config .dependency-cruiser.js "{libs,apps}"
yarn run v1.22.10
$ C:\p\monorepo\node_modules\.bin\depcruise --config .dependency-cruiser.js {libs,apps}

  ERROR: Extracting dependencies ran afoul of...

  Invalid escape sequence (243:438)
... in apps/web/assets/images/brands/logo.png
```

The file it crashes on goes to another file when I delete that one, so I'm not sure if it's all or not. My money is on not. I think it's a certain combination of bytes within them causing something to trip up. I might have to email you one, unless you can try some NASA ones or something. Let me know

🍻

**Comment by sverweij** [comment]
@binary64 thanks for the feedback. This is one of the rough edges. The most likely cause is that one of the parsers is trying to digest a .png as javascript and not liking it very much :-). Will be ironed out soon.

**Comment by sverweij** [comment]
In the coming days will be released in dependency-cruiser@10.3.0 (and/ or up). Heads ups will follow in this branch.
